voting_schemes/electionguard/python-wrapper/Pipfile

```diff
@@ -6,6 +6,7 @@ verify_ssl = true
 [dev-packages]
 flake8 = ">=3.8.3"
 pytest = "*"
+pytest-asyncio = "*"
 black = "*"
```

Comment:
Shouldn't we pin these versions just to be sure that this doesn't break in an update?

Reply:
but all the other dependencies aren't blocked to a version. This justs adds async library allowed by pyodide 17